lib/datadog/core/telemetry/heartbeat.rb

```diff
@@ -10,9 +10,7 @@ module Telemetry
       class Heartbeat < Core::Worker
         include Core::Workers::Polling
 
-        DEFAULT_INTERVAL_SECONDS = 60
-
-        def initialize(enabled: true, interval: DEFAULT_INTERVAL_SECONDS, &block)
+        def initialize(interval:, enabled: true, &block)
           # Workers::Polling settings
           self.enabled = enabled
           # Workers::IntervalLoop settings
```

Comment:
Minor: I suggest propagating the `_seconds` prefix internally as well:

(lol github doesn't allow me to suggest down to the next line ;_;)

```suggestion
        def initialize(heartbeat_interval_seconds:, enabled: true, &block)
          # Workers::Polling settings
          self.enabled = enabled
          # Workers::IntervalLoop settings
          interval = heartbeat_interval_seconds
```